spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -68,14 +68,18 @@ def create_call(user_params)
   end
 
   describe 'PUT #update' do
-    def update_user_call(user_params)
+    def update_user_call(user_params, headers)
+      request.headers.merge! headers
       put :update, params: { id: user.id, user: user_params }
     end
 
     context 'when is successful' do
       let(:user_params) { { email: 'email@dominio.com', password: '123456' } }
+      let(:headers) do
+        { 'Authorization': authenticate_user(user.id) }
+      end
```

Comment:
same comment about mehtods like **def <variable>**

Reply:
It will change with the `authenticated_user` method that Zorro recommends.